Mathlib/GroupTheory/SpecificGroups/Alternating/Centralizer.lean

```diff
@@ -96,7 +96,7 @@ theorem card_of_cycleType_mul_eq (m : Multiset ℕ) :
 variable (α) in
 /-- The cardinality of even permutations of given `cycleType` -/
 theorem card_of_cycleType (m : Multiset ℕ) :
-    (Finset.univ.filter fun g : alternatingGroup α => (g : Equiv.Perm α).cycleType = m).card =
+    ({g | (g : Equiv.Perm α).cycleType = m} : Finset (alternatingGroup α)).card =
```

Comment:
Does it work to use the # notation here too; then maybe you can avoid the type ascription?

Reply:
It does. Changed here and in some other places.